lexibank_abvdoceanic.py

```diff
@@ -65,6 +65,17 @@ def cmd_download(self, args):
 
 
     def cmd_makecldf(self, args):
+
+        # Load ignore list
+        ignore_list = {}
+        try:
```

Comment:
Try/Except should not be needed as long as you place the file there so I'd suggest to assume that the file is there.